src/Codegen/Constraints/ObjectBuilder.php

```diff
@@ -76,10 +76,10 @@ public function build(): this {
       ($additional_properties is nonnull || $pattern_properties is nonnull) &&
       $properties is nonnull
     ) {
-      $properties = $this->typed_schema['properties'] ?? [];
+      $properties = $this->typed_schema['properties'] ?? dict[];
       $hb = $this->getHackBuilder()
         ->addValue(
-          keyset(\HH\Lib\Vec\keys($properties)),
+          Keyset\keys($properties),
```

Comment:
Taking a `keyset()` of a `Vec\keys()` is the same thing as just taking a `Keyset\keys()`.